agent.go

```diff
@@ -967,6 +764,26 @@ func (a *Agent) Close() error {
 	return nil
 }
 
+// Cleanup all candidates
+func (a *Agent) flushCandidates() {
```

Comment:
Oh yea agree! What do you think of `purgeCandidates`? `clean` makes me think they are being sanitized, I do like `empty` though but a little ambiguous still. Like maybe you are emptying some state on the candidates.

Reply:
or `deleteAllCandidates`. I looked at the W3C docs, and it says that `restartIce` deletes existing candidates, so that terminology lines up with existing stuff.

Reply:
`deleteAllCandidates` sounds very clear to me.